src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseHelpAndSupportSection.tsx

```diff
@@ -30,31 +53,17 @@ const ReleaseHelpAndSupportSection = ({
           caption="Find out how and why we collect, process and publish these statistics"
           headingTag="h3"
         >
-          {publication.methodologies.length ||
-          publication.externalMethodology ? (
+          {allMethodologies.length ? (
             <>
-              {publication.methodologies.map(methodology => (
-                <p key={methodology.id} className="govuk-!-margin-bottom-9">
+              {allMethodologies.map(methodology => (
+                <p key={methodology.key} className="govuk-!-margin-bottom-9">
                   {editingMode === 'edit' ? (
                     <a>{`${methodology.title}`}</a>
                   ) : (
-                    <Link to={`/methodology/${methodology.id}/summary`}>
-                      {methodology.title}
-                    </Link>
+                    <Link to={methodology.url}>{methodology.title}</Link>
```

Comment:
Looks like this should be list rather than a bunch of paragraphs. The bottom margin is also too generous.

Reply:
I've done this now, thanks.